lib/lotus/config/assets.rb

```diff
@@ -48,11 +48,33 @@ def initialize(path)
         end
       end
 
+      class Sources
```

Comment:
@AlfonsoUceda This class is transitory. Assets sources will be handled at the level of Container, not single app. There is also a `method_missing` below that I accepted only with this transition in mind.